sqlite_query.c

```diff
@@ -325,3 +324,29 @@ static const char* sqlite_datatype(int t)
 			return azType[0];
 	}
 }
+
+/*
+ * Human readable message about disallowed combination of PostgreSQL columnn
+ * data type and SQLite data value affinity
+ */
+static void sqlite_value_to_pg_error (Oid pgtyp, int pgtypmod, sqlite3_stmt * stmt, AttrNumber attnum, int sqlite_value_affinity, int affinity_for_pg_column, int value_byte_size_blob_or_utf8)
+{
+	const char	*sqlite_affinity = 0;
+	const char	*pg_eqv_affinity = 0;
+	const char	*pg_dataTypeName = 0;
+	const int	 max_logged_byte_length = 63;
```

Comment:
Changed to famous constant `NAMEDATALEN`. This is my "too big for logging" limit, because identifiers in PostgreSQL are always logged and looks nice during log reading in terminal.

Reply:
Thank you for updating.